libs/components/src/lib/components/card/card.component.ts

```diff
@@ -1,25 +1,33 @@
-import { ChangeDetectionStrategy, Component, HostBinding, Input } from '@angular/core';
-import { prizmGetShadow, PrizmShadowDirective, PrizmShadowType } from '../../directives/shadow';
+import { ChangeDetectionStrategy, Component, HostBinding, Inject, Input, Optional } from '@angular/core';
+import { prizmGetShadow, PrizmShadowType } from '../../directives/shadow';
 import { PrizmShadowValue } from '../../directives/shadow/models';
 import { PrizmAbstractTestId } from '../../abstract/interactive';
 import { CommonModule } from '@angular/common';
+import { PRIZM_CARD_OPTIONS, prizmCardDefaultOptions, PrizmCardOptions } from './card-options';
 
 @Component({
   selector: 'prizm-card',
   templateUrl: './card.component.html',
   styleUrls: ['./card.component.less'],
   changeDetection: ChangeDetectionStrategy.OnPush,
-  providers: [],
   standalone: true,
-  imports: [CommonModule, PrizmShadowDirective],
+  imports: [CommonModule],
 })
 export class PrizmCardComponent extends PrizmAbstractTestId {
-  @Input() shadow: PrizmShadowType = 'mini-bottom';
+  @Input() shadow: PrizmShadowType | undefined;
+
+  override readonly testId_ = 'ui_card';
+
+  private options: PrizmCardOptions = { ...prizmCardDefaultOptions };
+
+  constructor(@Optional() @Inject(PRIZM_CARD_OPTIONS) customOptions: PrizmCardOptions) {
```

Comment:
указанный тип не соответствует используемым декораторам:
```suggestion
  constructor(@Optional() @Inject(PRIZM_CARD_OPTIONS) customOptions: Partial<PrizmCardOptions> | null) {
```

Reply:
спасибо, partial тут правда упустила, null считаю, что не нужно.

Reply:
null - [это для согласованности с `@Optional`](https://github.com/angular/angular/blob/412ac303bc926d8e929361cc776505bb0d8b4da4/packages/core/src/di/injector_compatibility.ts#L155) - но да, в рантайме все ок, т.к. `{ ...null } == {}`

Reply:
по идее и TS при строгих настройках это должен требовать, но я ни разу не видела такого на практике, видимо обычно руководствуются идеей, что @optional уже подразумевает возможность появления null (по аналогии с необязательными членами класса?). 
 я за единообразие - в коде проекта везде используется optional без указания возможности null. поэтому выбор был в эту сторону. 
 
еще посмотрела в material - там аналогично не указывают явно null. попробую поискать конкретные рекомендации по этому поводу. Если у тебя что-то такое будет, скинь пожалуйста? очень любопытная тема.